wemake_python_styleguide/visitors/ast/blocks.py

```diff
@@ -76,7 +76,7 @@ class BlockVariableVisitor(base.BaseNodeVisitor):
     )
 
     _scope_predicates: Tuple[_ScopePredicate, ...] = (
-        lambda node, names: predicates.is_property_setter(node),
+        lambda node, names: predicates.is_property_setter(node),  # noqa: WPS467
```

Comment:
This should be ignored, because here we assign `Tuple` and not `lambda`.

We should test it explicitly.